charts/elasticsearch-index/values.yaml

```diff
@@ -3,8 +3,10 @@ elasticsearch:
   #host:  # elasticsearch-client|elasticsearch-ephemeral
   port: 9200
   index: directory
-hook:
-# when hook.enabled=true (default), this chart does not work standalone, but is intended as
-# either a 'post'-subchart to an elasticsearch chart or a 'pre'-subchart to wire-server
-  enabled: true
-  type: pre # possible values: 'pre' or 'post'
+cassandra:
+  # host:
+  port: 9042
+  keyspace: brig
+image:
+  repository: quay.io/wire/brig-index
+  tag: 2.77.11
```

Comment:
We can now remove this tag, right?

Reply:
I think it would get overwritten when we merge wire-server. Until then, this is a working build

Reply:
Or maybe comment it out and leave a comment that: _it must be at least version 2.77.11_ ?

Reply:
Hmm, let me see if cailieach will like a comment there. I think we do some sed magic to bump versions. Why is it a problem if we have this version here?

Reply:
My bad, it wasn't caillieach, it was [set-version.sh](https://github.com/wireapp/wire-server-deploy/blob/3bc2db80c2cb05917c82333e53bbf7b0ba470878/bin/set-version.sh#L15) in this repo, we rely on that for the version to keep ticking. So, we can leave the make it `version: ` and leave a comment above. But I am not sure why we would go through the hassle.

Reply:
> Hmm, let me see if cailieach will like a comment there. I think we do some sed magic to bump versions. Why is it a problem if we have this version here?

Hmmm, I think you are right. It will get overwritten in the CI process. I just wanted to make sure to _not_ pin upcoming charts (on develop and master) to this version.